components/org.wso2.micro.integrator.extensions/org.wso2.micro.integrator.management.apis/src/main/java/org/wso2/micro/integrator/management/apis/UserResource.java

```diff
@@ -180,4 +188,21 @@ private boolean isAdmin(String[] rolesList) throws UserStoreException {
         return Arrays.asList(rolesList).contains(getRealmConfiguration().getAdminRoleName());
     }
 
+    protected Boolean isValidUserStore() {
+
+        Object fileUserStore = ConfigParser.getParsedConfigs().get("internal_apis.file_user_store.enable");
```

Comment:
Can we use https://github.com/wso2/micro-integrator/blob/b6f301980b33d742918a7683b4e62bab6cbeb280/components/org.wso2.micro.integrator.extensions/org.wso2.micro.integrator.management.apis/src/main/java/org/wso2/micro/integrator/management/apis/security/handler/SecurityHandlerAdapter.java#L44 instead? then it'll be consistent

Reply:
changed as discussed